### ISSUE TITLE
angular 1.3?

### ISSUE THREAD

**Issue by mg1075** [cdjackson/angular-dygraphs#9]
Has this directive been tested against angular 1.3? I am getting a race condition on 1.3 with the "resize()" function.

**Comment by mg1075** [comment]
The race condition went away when I styled the height **outside** of the directive.

**Comment by cdjackson** [comment]
Yes - its working fine here under 1.3 and the github demo is using 1.3.6 and that also works fine.
